modular_skyrat/modules/novaya_ert/code/rationpacks.dm

```diff
@@ -1,55 +1,176 @@
+	foodtypes = GRAIN | FRIED | DAIRY
+
+/obj/item/food/colonial_course/blins
+	name = "condensed milk crepes"
+	desc = "A vacuum-sealed four-pack of stuffed crepes with a minimal amount of markings. There is nothing else to it, to be frank.\
+	<br> Surprisingly tasty for its looks, as long as you're not lactose intolerant, on diet, or vegan. The back of the packaging is covered in a mass of information detailing the product."
+	food_reagents = list(/datum/reagent/consumable/nutriment = 2,
+						/datum/reagent/consumable/caramel = 3,
+						/datum/reagent/consumable/milk = 4)
+	trash_type = /obj/item/trash/blins
+	icon_state = "blin"
+	tastes = list("insane amount of sweetness" = 10, "crepes" = 3)
+	foodtypes = SUGAR | GRAIN | DAIRY | BREAKFAST
+
+/obj/item/reagent_containers/cup/glass/coffee/colonial
+	name = "colonial thermocup"
+	desc = "Technically, used to drink hot beverages. But since it's the only cup design that was available, you gotta make do."
+	special_desc = "A small instruction on the side reads: <i>\"For use in food replicators; mix water and powdered solutions in one-to-one proportions.</i>"
+	icon = 'modular_skyrat/modules/novaya_ert/icons/rationpack.dmi'
+	list_reagents = list(/datum/reagent/consumable/powdered_coffee = 25)
+
+/obj/item/reagent_containers/cup/glass/coffee/colonial/empty
+	list_reagents = null
+
+/obj/item/trash/pljeskavica
+	name = "pljeskavica wrapping paper"
+	desc = "Covered in sauce smearings and smaller pieces of the dish on the inside, crumpled into a ball. It's probably best to dispose of it."
+	icon = 'modular_skyrat/modules/novaya_ert/icons/rationpack.dmi'
+	icon_state = "borgir_trash"
+
+/obj/item/trash/nachos
+	name = "empty nachos tray"
+	desc = "Covered in sauce smearings and smaller pieces of the dish on the inside, a plastic food tray with not much use anymore. It's probably best to dispose of it or recycle it."
+	icon = 'modular_skyrat/modules/novaya_ert/icons/rationpack.dmi'
+	custom_materials = list(/datum/material/plastic = HALF_SHEET_MATERIAL_AMOUNT)
+	icon_state = "nacho_trash"
 
-/obj/item/storage/box/nri_rations
-	name = "NRI combat rations"
-	desc = "A packaged military MRE for soldiers in extended deployments and people who can deal with the shits after finishing one. Everything on the back and front is written in Pan-Slavic, but you can make out 'Једите пре: Сентябрь 2603'."
+/obj/item/trash/blins
+	name = "empty crepes wrapper"
+	desc = "Empty torn wrapper that used to hold something ridiculously sweet. It's probably best to recycle it."
+	icon = 'modular_skyrat/modules/novaya_ert/icons/rationpack.dmi'
+	custom_materials = list(/datum/material/plastic = HALF_SHEET_MATERIAL_AMOUNT*0.5)
+	icon_state = "blin_trash"
+
+/obj/item/storage/box/gum/colonial
+	name = "mixed bubblegum packet"
+	desc = "The packaging is entirely, -finally-, is written in something intelligible: Pan-Slavic, with a small text of Sol Common. That can be examined."
```

Comment:
```suggestion
	desc = "The packaging is entirely written in Pan-Slavic, with a small blurb of Sol Common. You would need to take a better look to read it, though, as it is written quite small."
```

Reply:
Initially I took the original gum package's description and edited it minimally; your suggestion is better tbh, will use that instead.